### PR TITLE
PSP-3726 tests

### PR DESCRIPTION
**Pull request by asanchezr**

### CODE REVIEW

File: backend/api/Services/NoteService.cs [hunk 1]
```diff
@@ -93,6 +94,7 @@ public NoteModel Update(NoteModel model)
         public void DeleteNote(NoteType type, long noteId)
         {
             this.Logger.LogInformation("Deleting note with type {type} and id {noteId}", type, noteId);
+            this.User.ThrowIfNotAuthorized(Permissions.NoteDelete);
```
Comment (asanchezr) [inline backend/api/Services/NoteService.cs:97]: Claims check was missing from original implementation

File: backend/api/Services/NoteService.cs [hunk 2]
```diff
@@ -117,6 +119,7 @@ public void DeleteNote(NoteType type, long noteId)
         public IEnumerable<PimsNote> GetNotes(NoteType type, long entityId)
         {
             this.Logger.LogInformation("Getting all notes with type {type} and parent id {entityId}", type, entityId);
+            this.User.ThrowIfNotAuthorized(Permissions.NoteView);
```
Comment (asanchezr) [inline backend/api/Services/NoteService.cs:122]: Claims check was missing from original implementation

File: backend/dal/Repositories/NoteRepository.cs [hunk 3]
```diff
@@ -91,7 +91,7 @@ public int Count()
         public IEnumerable<PimsNote> GetActivityNotes(long entityId)
         {
             return this.Context.PimsActivityInstanceNotes
-                .Where(x => x.ActivityInstanceId == entityId && x.IsDisabled == false).Select(x => x.Note).ToList();
+                .Where(x => x.ActivityInstanceId == entityId && (x.IsDisabled ?? false) == false).Select(x => x.Note).ToList();
```
Comment (asanchezr) [inline backend/dal/Repositories/NoteRepository.cs:94]: IsDisabled is nullable to I accounted for null - wasn't done originally

File: backend/api/Services/NoteService.cs [hunk 4]
```diff
@@ -71,10 +71,11 @@ public EntityNoteModel Add(NoteType type, EntityNoteModel model)
 
         public NoteModel Update(NoteModel model)
         {
-            this.Logger.LogInformation("Updating note with id {id}", model.Id);
```
Comment (devinleighsmith) [inline backend/api/Services/NoteService.cs:74]: ah, yes, good, avoid the npe.

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**